internal/connectors/db/process_result_set.go

```diff
@@ -94,30 +110,37 @@ func ReadOperationFromResultSet(res result.Result) (types.Operation, error) {
 		operationState = types.OperationState(*operationStateBuf)
 	}
 	if operationType == string(types.OperationTypeTB) {
-		if backupId == nil || database == nil || ydbOperationId == nil {
+		if backupId == nil || databaseName == nil || databaseEndpoint == nil || ydbOperationId == nil {
```

Comment:
I think we should specify it in scheme as NOT NULL. But it is for next pr i guess